action.php

```diff
@@ -295,12 +296,16 @@ public function handleActPreprocessTranslate($event, $param) {
             return;
         }
 
+        // Use suggestion when not specified.
+        if (empty($target_id)) {
+            $target_id = $my->suggestTranslationId($target_title, $target_lang, $source_lang);
+        }
+        
         // Check if target page exists
-        $target_id = $my->suggestTranslationId($target_title, $target_lang, $source_lang);
         if (page_exists($target_id)) {
             // Error message
             //$this->_formErrors['title'] = 1;
-            msg(sprintf($this->getLang['e_pageexists'], $target_title),-1);
+            msg(sprintf($this->getLang('e_pageexists'), $target_id),-1);
```

Comment:
This is the error you wanted to fix, right? The diff has a lot of other changes, similar to your other PR. If you fix getLang only and revert everything else, i can merge.